### PR TITLE
Broken links

### PR DESCRIPTION
**Pull request by jglick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 Current results in summary:
 * 1114 plugins
 * 560 plugins using a deprecated Jenkins api
 * 20 deprecated classes, 170 deprecated methods and 12 deprecated fields are used in plugins
 * 28 deprecated and public Jenkins classes, 319 deprecated methods, 58 deprecated fields are not used in the latest published plugins
 
-See details and deprecated usage for each plugin in the [continuous integration](https://ci.jenkins-ci.org/view/All/job/Reporting/job/infra_deprecated-usage-in-plugins/lastSuccessfulBuild/artifact/target/output.html) or in this [example of output](../../blob/master/Output_example.html).
+See details and deprecated usage for each plugin in the [continuous integration](https://ci.jenkins.io/job/Infra/job/deprecated-usage-in-plugins/job/master/lastSuccessfulBuild/artifact/output/) or in this [example of output](../../blob/master/Output_example.html).
+
+<!-- or https://ci.jenkins.io/job/Reporting/job/deprecated-usage-in-plugins/ ? -->
```
Comment (jglick) [inline README.md]: @abayer says this one should get deleted.

**Comment by jglick** [comment]
ping @daniel-beck

**Comment by batmat** [comment]
Now conflicted with the link fix in master.

**Comment by jglick** [comment]
From @batmat’s #10, but that just leaves open the question of why we have both https://ci.jenkins.io/job/Infra/job/deprecated-usage-in-plugins/job/master/ and https://ci.jenkins.io/job/Reporting/job/deprecated-usage-in-plugins/ performing builds.

**Comment by batmat** [comment]
Asked on IRC, we'll see.

**Pull request closed without merging**